cpp/src/aztec3/circuits/rollup/base/utils.cpp

```diff
@@ -1,9 +1,14 @@
+#include "aztec3/circuits/rollup/base/nullifier_tree_testing_harness.hpp"
+#include "aztec3/circuits/rollup/base/utils.hpp"
 #include "aztec3/constants.hpp"
 #include "index.hpp"
 #include "init.hpp"
 
 #include <aztec3/circuits/kernel/private/utils.hpp>
 #include <aztec3/circuits/mock/mock_kernel_circuit.hpp>
+#include <cstddef>
+#include <cstdint>
+#include <sys/types.h>
```

Comment:
What functionality were these needed for?

Reply:
they arent :o removing